unconference/LMIC-know-how.md

```diff
@@ -0,0 +1,200 @@
+
+## Learning modules
+
+### Module 1: Scope of the curriculum - who is this resource for and how to use it
+
+* Why are we doing this (mission statement)
+* How these materials can be used (teaching material), licensing (CC-BY-NC), attribution
+* Learner personas
+* Overview of problems when applying for grants (challenges these materials will address)
+    * The grant does not fit the remit of the call for proposals
+    * Writing and grammar skills
+    * Persons/Collaborators do not have the skills
+    * Unrealistic timeline or budget
+    * Application does not show experience with grant writing
+* What this material will NOT teach
+    * Comprehensive overview of grant writing (there are other grant writing workshops, and this won’t cover discipline-specific considerations)
+    * Complete information about program/project management and leadership (community management, managing personnel), and planning for organizational sustainability
+
+*Module 1 learning objectives:* 
+At the end of this module, learners should be able to:
+* Understand the scope of this curriculum
+* Understand how to use and reuse these materials 
+* Recognize the target the audience of this content
+* Identify common issues while seeking funding that this curriculum will address
+
+
+### Module 2: Preparing to apply for grants
+
+* What is a grant and what are the types of grants
+    * Competitive requests for applications (RFAs)
+    * Unsolicited submissions and pitching ideas
+* Meeting and getting to know funders (building trust, learning about program priorities)
+* Building a professional brand
+    * Website for lab, organization, or individuals
+    * Basic information about goals
+* Reading the RFA
+    * How the grant will be evaluated
+    * Frame your proposal around the evaluation rubrics
+* What do funders look for in candidates 
+* Organizational requirements
+    * Connecting with your institution and learning about procedures
+    * Finding or learning if you need to create an organization to support you (fiscal sponsorship)
+* Finding supporters that will give you help/feedback for your proposal
+* Templates
+    * Collaborator recommendation letters 
+    * CVs
+
+*Module 2 learning objectives:* 
+At the end of this module, learners should be able to:
+* Identify grant opportunities for which they are eligible
+* Identify key collaborators or stakeholders that need to be involved
+* Understand the basic requirements you need to meet for a grant application
+* Decypher the RFA specifics and translate them into actionable application tasks
+
+
+### Module 3: Preparing a proposal
+
+* Preparing a proposal
```

Comment:
comment: these sections are often very funder-specific; some funders don't ask for a DEI statement; most funders do ask for information about the PIs (biosketch, CV, etc.) and this might be worth calling out here.